v2/emailpassword/migration/about.mdx

```diff
@@ -9,42 +9,27 @@ hide_title: true
 
 # About
 
-In this guide we will be going through the process of migrating users from an external Authentication provider to SuperTokens.
+This guide explains how to migrate users from an external authentication provider to SuperTokens.
```

Comment:
migrate users from a different authentication provider such as `Auth0`, ... etc to SuperTokens.